wazuh/wazuh-splunk
# Rework of the MITRE alerts view

## Summary

Closes #1234 

Due to the extension of the MITRE Framework on Wazuh, this section had to be rework in order to provide the new functionalities.

![mitre-splunk](https://user-images.githubusercontent.com/15186973/166008107-3e92dd12-e8ad-4264-b6bd-2b6bf61a7355.gif)

## To test

- Go to the `MITRE ATT&CK` module on the Overview section.
- Click on `Show alerts`
- Check that the alerts correspond to the alerts shown on the Dashboard.
- Check that clicking on each MITRE Tecnhique opens a model with informatin about it.
- Check that no error happens.
- Check that information shown makes sense.
- Note that some MITRE Technique details are not provided yet by the API, so **N/A** or **No description provided.** is shown instead.
